x-pack/packages/kbn-elastic-assistant/impl/connectorland/use_load_connectors/index.tsx

```diff
@@ -44,7 +45,7 @@ export const useLoadConnectors = ({
         (acc: AIConnector[], connector) => [
           ...acc,
           ...(!connector.isMissingSecrets &&
-          [actionTypeKey.bedrock, actionTypeKey.openai].includes(connector.actionTypeId)
+          [actionTypeKey.bedrock, actionTypeKey.openai, actionTypeKey.gemini].includes(connector.actionTypeId)
```

Comment:
I think we should hold off on adding the connector to the assistant in this PR as it will not work until your next PR

Reply:
done